src/com_tjvendors/site/includes/utilities.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  com_tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2020 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die();
+use Joomla\CMS\Table\Table;
+
+Table::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_tjfields/tables');
```

Comment:
@deepgawade89 Add this where required. Because in future versions, utilities in going to be used in overall TJVendor